src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -403,6 +457,16 @@ export const _handleFormAuthRedirect: ControllerHandler<
             return SgidService.createRedirectUrl(
               formId,
               Boolean(isPersistentLogin),
+              [],
+              encodedQuery,
+            )
+          })
+        case FormAuthType.SGID_MyInfo:
+          return validateSgidForm(form).andThen(() => {
+            return SgidService.createRedirectUrl(
+              formId,
+              Boolean(isPersistentLogin),
```

Comment:
ref our discussion yesterday, are we still able to support isPersistentLogin if we are now storing only the access token in jwt (vs the userinfo), and exchanging for userInfo upon loading the form?

Reply:
Yup! This is because the cookie is dual-use, ref https://github.com/opengovsg/FormSG/pull/6337#discussion_r1197378671.

Reply:
^ as above. with dual-use cookie, will the persistent login be cleared if user then logs into a sgid-myinfo form (as the dual use cookie will be overwritten to store sgID access token)